### PR TITLE
 Icon for toolbar button "user"

### PR DESCRIPTION
**Pull request by abhinaisai2002**
Fix issue : #5119 
Changed the svg of toolbar icon to gear svg icon.

### CODE REVIEW

**Comment by abhinaisai2002** [comment]
@stevepiercy can you help me in this, I already signed the document

File: news/5109.feature
Comment (stevepiercy) [inline news/5109.feature:1]: Please restore this file. Do not delete files from this directory. They are used to generate the change log entries by the release manager. Instead in your local repo, you should pull `master`, then merge `master` to your local branch, and finally push your local branch to GitHub.

File: news/5119.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+changed the icon of toolbar to gear icon.
```
Comment (stevepiercy) [inline news/5119.bugfix:1]: Always use proper English grammar and syntax, and give yourself credit in news items.

```suggestion
Changed the icon of toolbar to gear icon. @abhinaisai2002
```

File: src/icons/user.svg
Comment (stevepiercy) [inline src/icons/user.svg:1]: Do not delete icons from this directory. Please restore it. They may be used elsewhere in the UI. See my comment at:

https://github.com/plone/volto/issues/5119#issuecomment-1722841188

What is the source of the gear icon? If you grabbed it from Twitter Bootstrap, do we have a license to use it? We can get in huge trouble with third parties for not following their license.

We already have an icon that looks like a gear, so you do not need to add one from wherever you got it.

Finally I don't think the issue is ready for development, and will require further discussion before any action is taken. Sorry that was not clear.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
@abhinaisai2002 per https://6.docs.plone.org/contributing/index.html

> After a member of the Plone Foundation reviews and accepts your signed agreement, your GitHub account will be added to a team in the Plone GitHub organization with appropriate access. This process may take a few business days.

I see that you are now a Member of plone/contributors and Plone Foundation in you GitHub user profile.

What else do you need help with?

**Comment by abhinaisai2002** [comment]
@stevepiercy while raising a pr, the workflow is showing a error on contributor document

**Comment by stevepiercy** [comment]
@abhinaisai2002 that is correct behavior. At the time you pushed the last commit, you were not approved as a member. Once you push new commits, whether in this PR or another, then the message will no longer recur.

**Comment by Wagner3UB** [comment]
@abhinaisai2002 Are you sure you are working on the right issue here? The Sharing component has no relation to the users button on the sidebar.

**Pull request closed without merging**